packages/wonder-blocks-form/components/checkbox-core.js

```diff
@@ -0,0 +1,225 @@
+// @flow
+// The internal stateless ☑️ Checkbox.
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+
+import Color, {mix, fade} from "@khanacademy/wonder-blocks-color";
+import {View, addStyle} from "@khanacademy/wonder-blocks-core";
+import Icon from "@khanacademy/wonder-blocks-icon";
+
+import type {ClickableHandlers} from "@khanacademy/wonder-blocks-core";
+import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
+
+import {getBackgroundColor} from "../util/util.js";
+import type {SharedProps} from "../util/types.js";
+
+type Props = {|
+    ...SharedProps,
+    ...ClickableHandlers,
+    hovered: boolean,
+    focused: boolean,
+    pressed: boolean,
+|};
+
+const {blue, red, white, offWhite, offBlack16, offBlack32, offBlack50} = Color;
+
+const StyledInput = addStyle("input");
+
+const checkboxCheck: IconAsset = {
```

Comment:
Err - so all checkboxes supposed to have the smaller check, not just the ones in the menu? I feel like I'd super-like the menu checkbox to use this checkbox-core.js now, if at all possible (even if we have a separate set of focus/hover states)!